src/components/dialogmote/referat/Referat.tsx

```diff
@@ -23,8 +30,10 @@ const texts = {
     "Du må aldri skrive sensitive opplysninger om helse, diagnose, behandling, og prognose. Dette gjelder også hvis arbeidstakeren er åpen om helsen og snakket om den i møtet.",
 };
 
-interface ReferatSkjemaValues {
-  deltakere: Deltaker[];
+export interface ReferatSkjemaValues {
+  deltakerArbeidstaker: boolean;
```

Comment:
Endret til å bruke tre skjema-felter (ett for hver deltaker) med true/false verdi i steden for en Deltaker-array for å enklere knytte feilmelding til riktig sjekkboks.